### ISSUE TITLE
General Midi sounds different

### ISSUE THREAD

**Issue by Zwarteziel** [AshLancer/QFG4-Enhanced#3]
Hi,

thank you for making these series of patches. I was exited to start playing one of my favourite entries in the series once more. For music, I'm using a (real) Sound Canvas. (Game is set to use 'General Midi' for music). I notice a big difference in the song that plays during the intro / main menu. It seems as if one or more of the instruments have been replaced by others. Is there any way to revert to the original midi music?

**Comment by Zwarteziel** [comment]
A follow-up: this morning I checked to see which .scr and .hep files were present in both my "original" (GOG) version and the new patch. I overwrote them one by one, each time checking if the midi-music was different. Strangely - but luckily I guess - the game now sounds fine. I'm quite sure I didn't mess with instrument mappings before trying the different versions yesterday, but well... It seems the game is functioning. I'll close this issue and thank you for your efforts!

For the record, these were the files I swapped manually:

- 26
- 250
- 500
- 580
- 641
- 710
- 730
- 731

**Comment by TarkinMX** [comment]
The directions for installing the patch are somewhat confusing and I had this same issue. It should state rather than renaming patches to patches_bak, to make a copy of the patches folder and then rename that to patches_bak. With the original patches folder still present and unaltered, copy everything from this patch into that folder and overwrite everything in that folder.

**Comment by AshLancer** [comment]
Hi,

I just wanted to comment: I haven't changed any of the midi music in the game. All of the script patch files only contain game code, with a couple of views (sprites) and MSG (text). Sorry to hear that the directions were a bit confusing. I tried to make it as simple as possible. Will do better next time. Thank you for playing, and please feel free to point these things out!

**Comment by TarkinMX** [comment]
I think the issue he had was the same I had. I read the instructions, renamed the PATCHES folder to PATCHES_BAK and then moved the PATCHES folder in your zip to the game folder. It caused me to lose the same audio with the wolf sound that he described. Looking at the instructions, I realized that I never got a prompt to overwrite anything so I deleted the PATCHES folder, renamed PATCHES_BAK to PATCHES and then made a copy of it to PATCHES_BAK. Once I copied the folder from the zip again and overwrote everything, it was working correctly.
